surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/fixture/HelperAssertions.java

```diff
@@ -159,4 +164,10 @@ public static IntegrationTestSuiteResults parseReportList( List<ReportTestSuite>
             throw new RuntimeException( "Couldn't parse XML reports", e );
         }
     }
+
+    public static void assumeJavaVersion(JavaVersion version)
+    {
+        assumeThat( "java.specification.version: ",
+                SystemUtils.JAVA_SPECIFICATION_VERSION, is( greaterThanOrEqualTo( version.toString() ) ) );
```

Comment:
Hello @Tibor17 

> Unfortunately org.apache.commons.lang3.JavaVersion is not Comparable

Java enums are comparable by default.

> How would this work with string 1.10? [...]

I'm currently working on Commons Lang Release 3.5 which I expect to be ready in two weeks or so. Commons Lang 3.5 will have full Java 9 support with regards to version number. Furthermore it will have a better API for comparing the current version against a desired version:

```
if(JavaVersion.CURRENT.atLeast(JavaVersion.JAVA_1_8)) {
   // do something only possible with Java 8
}
```

I suggest to apply this for now so I can finish the junit-5 IT PR with this new check API. I will provide a PR for updating this again to the new API introduced in Commons Lang 3.5 when Lang 3.5 is ready. Makes sense?